flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/tasks/ChainingOutput.java

```diff
@@ -129,13 +126,7 @@ public void emitLatencyMarker(LatencyMarker latencyMarker) {
 
     @Override
     public void close() {
-        try {
-            if (closeable != null) {
-                closeable.close();
-            }
-        } catch (Exception e) {
-            throw new ExceptionInChainedOperatorException(e);
-        }
```

Comment:
unit test?

Reply:
I don't think, I need to test an empty method. I have added a comment that only things that are owned should be closed with a reference to this ticket to protect against regressions.